metrics.go

```diff
@@ -60,6 +73,13 @@ func newConsumerMetrics(consumerName, prefix string) *ConsumerMetrics {
 	kafkaMetrics.wmsBatchDurationTimer = metrics.NewRegisteredTimer(fmt.Sprintf("%sWMsBatchDuration-%s", prefix, consumerName), kafkaMetrics.registry)
 	kafkaMetrics.wmsIdleTimer = metrics.NewRegisteredTimer(fmt.Sprintf("%sWMsIdleTime-%s", prefix, consumerName), kafkaMetrics.registry)
 
+	kafkaMetrics.numFetchedMessagesCounter = metrics.NewRegisteredCounter(fmt.Sprintf("%sFetchedMessages-%s", prefix, consumerName), kafkaMetrics.registry)
```

Comment:
Can we do something here where our go doc can be more expressive. with that details on the different metrics and such? Understanding what each of these do is that defined somewhere? Let me know I can write it up but want to go through what each and everyone of these means and do and if we are missing any still too. more metrics!!!!